Sources/Sentry/Public/SentryOptions.h

```diff
@@ -187,6 +187,13 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL enableNetworkTracking;
 
+/**
+ * THIS FLAG IS EXPERIMENTAL.
+ * When enabled, the SDK tracks performance for IO
+ * reads and writes if auto performance tracking is enabled. The default is <code>NO</code>
+ */
```

Comment:
`l`: I think we should mention it's only working for NSData.
```suggestion
 * This feature is EXPERIMENTAL.
 *
 * When enabled, the SDK tracks performance for file IO reads and writes with NSData if auto
 * performance tracking and enableSwizzling are enabled. The default is <code>NO</code>.
 */
```